query-engine/connector-test-kit-rs/query-engine-tests/tests/new/invalid_input_error.rs

```diff
@@ -0,0 +1,81 @@
+use query_engine_tests::*;
+
+/*
```

Comment:
Please do not leave commented out tests to code that gets merged to main. I've seen some of these just being commented out for years...

Reply:
This one is fair - it is a hack to be able to merge this now instead of having to fix the test setup. We should have an issue for the problem though so this can be prioritized (and then the issue mentioned in the comment)

Reply:
I have saved the commented-out code in a gist, and removed it from this PR as Julius suggested. Will add it back once we solve the test setup.